docs/features/networking.md

```diff
@@ -75,3 +75,15 @@ For example, here we construct an HTTP URL for our local web application and tel
 <!--codeinclude-->
 [Accessing the exposed host port from a container](../examples/src/test/java/generic/HostPortExposedTest.java) inside_block:useHostExposedPort
 <!--/codeinclude-->
+
+
+## Advanced networking
+
+Docker provides the ability for you to create custom networks and place containers on one or more networks. Then, communication can occur between networked containers without the need of exposing ports through the host. With Testcontainers, you can do this as well. 
+
+!!! warning
+    Note that Testcontainers currently only allows a container to be on a single network.
+
+<!--codeinclude-->
+[Creating custom networks](../examples/src/test/java/generic/ContainerNetworkCreationTest.java) inside_block:useCustomNetwork
```

Comment:
I wonder if you can refer to the `NetworksTest` from `../core/src/test`, could you please try it?

Reply:
I can certainly try it. Main reason I created a new example was that the tests either used rules or try-with blocks that made the example more cluttered in the doc. Have a preference on which to use?

Reply:
I think the rules-based one is pretty clean and understandable :)

Reply:
Alrighty... I went with the try-with block so it's not framework specific.